docs/rules/prefer-node-protocol.md

```diff
@@ -11,6 +11,12 @@ When importing builtin modules, it's better to use the [`node:` protocol](https:
 
 And don't forget to [upvote this issue](https://github.com/nodejs/node/issues/38343) if you agree.
 
+Note that this feature is not available in early versions of Node 12 and Node 14:
+
+> v16.0.0, v14.18.0 (ESM import and CommonJS require())
```

Comment:
```suggestion
> v16.0.0, v14.18.0 (CommonJS `require()`)
```

Not clear to me why this is mentioning ESM.

Reply:
Removed unnecessary mentions of ESM/CJS.